### ISSUE TITLE
Missing <math.h> include on MacOS

### ISSUE THREAD

**Issue by abroekhuis** [JamesBremner/raven-cSpline#4]
In the IsInputSane method, the "fabs" method is used, on MacOS this method is define in <math.h>. 
Adding #include <math.h> to the top of the file solves the issue.

**Comment by JamesBremner** [comment]
Are you sure?  Usually, it is in <cmath>.  http://www.cplusplus.com/reference/cmath/fabs/

**Comment by abroekhuis** [comment]
Yes, it is, but as shown on that page, it needs to be included via "math.h", and cspline.cpp doesn't include "math.h". 
I don't have a Linux setup atm, but I suspect that "math.h" is included via one of the other header files on Linux, but not on MacOS.

**Comment by JamesBremner** [comment]
I have neither Linux nor MacOS ( only Windows ), so I cannot advise you.

If you wish to add something like this to` cspline.cpp`

```C++
#ifdef _macos_
#include ( whatever you need )
#endif _macos_
```

and then submit a pull request, I will merge your contribution.
